src/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/PrimitiveGhostData.hpp

```diff
@@ -43,4 +47,48 @@ class PrimitiveGhostDataOnSubcells {
   static Variables<prims_to_reconstruct_tags> apply(
       const Variables<hydro::grmhd_tags<DataVector>>& prims) noexcept;
 };
+
+/*!
+ * \brief Projects the rest mass density \f$\rho\f$, pressure \f$p\f$,
+ * \f$W v^i\f$, magnetic field \f$B^i\f$, and the divergence cleaning field
+ * \f$\Phi\f$ so they can be projected to the subcells and sent to neighbors for
+ * subcell reconstruction.
+ *
+ * The computation copies the data from the primitive variables to a new
+ * Variables and computes \f$W v^i\f$, then does the projection. In the future
+ * we will likely want to elide this copy but that requires support from the
+ * actions. We should also consider if we want to use \f$u_i\f$ as a primitive
+ * instead of \f$v^i\f$ to further reduce work. It might also work to
+ * reconstruct \f$W v^i\f$ since this also guarantees a positive Lorentz factor
```

Comment:
Remaining confusion: "It might also work to reconstruct W v^i..." ... but isn't that what is being done? why phrased as a hypothetical? You also say (in the previous sentence) that u_i would be less work, and then in this sentence that v^i is cheaper to compute. This sounds either inconsistent or overly simplified.. but has me confused either way..

(Note: I do think it's great to document these avenues for future exploration! I just want the suggestions to be very clear so the person revisiting this doesn't go down the wrong path...)

Reply:
Good catch :) I've deleted the extra info since that change is already implemented :)